tools/torch_quant/torch_quant/quantizer.py

```diff
@@ -264,7 +264,7 @@ def qat_gm(
             # Generally we do not add fake-quant to bias during qat fine-tuning. If
             # users want to evaluate the accuracy of a model in specific state, they
             # should use the model returned by `quantizer.quantize`
-            quantizable_module_to_observed])
+            partial(quantizable_module_to_observed, is_observe_bias=False)])
```

Comment:
For qat, `is_observe_bias` is always `False`, `bias_ob_ctr` is no longer needed for `qat()`?

Reply:
For now, adding fake-quant for bias will slow down the training process and the benefits of doing so is unknown. So the fake-quant is removed. If any benefits is demonstrated, we can enable this.